R/bump-version.R

```diff
@@ -21,7 +21,7 @@ bump_version_impl <- function(which, no_change_behavior) {
     }
   }
   #' 1. [update_news()], using the `which` argument
-  update_news(which = which)
+  update_news(which = which, fledgeling = fledgeling)
```

Comment:
If this is the only reason, we should:

```suggestion
  update_news_impl(default_commit_range(), which = which, fledgeling = fledgeling)
```

or perhaps even change the signature of `update_news_impl()` to accept `commits` instead of `messages` .